src/aave-v3/SupplyVault.sol

```diff
@@ -215,15 +215,15 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
             uint256 claimedAmount = claimedAmounts[i];
             uint256 rewardsIndexMem = rewardsIndex[rewardToken];
 
-            if (supply > 0 && claimedAmount > 0) {
+            if (claimedAmount > 0) {
```

Comment:
Hmm you make the assumption that if `claimedAmount == 0` then `supply == 0` here, right?

Reply:
https://github.com/morpho-dao/morpho-tokenized-vaults/pull/223#discussion_r1031167190

Reply:
Works for me